packages/data-context/src/data/ProjectConfigIpc.ts

```diff
@@ -96,7 +107,8 @@ export class ProjectConfigIpc extends EventEmitter {
   }
 
   on(evt: 'childProcess:unhandledError', listener: (err: CypressError) => void): this
-
+  on(evt: 'export:telemetry', listener: (data: string) => void): this
+  on(evt: 'main:process:will:disconnect:ack', listener: () => void): this
```

Comment:
I don't think anything is happening with the return value?
```suggestion
  on(evt: 'main:process:will:disconnect:ack', listener: () => void): void
```